lib/db/parsed-statement.js

```diff
@@ -22,53 +21,10 @@ function ParsedStatement(type, operation, collection, raw) {
 }
 
 ParsedStatement.prototype.recordMetrics = function recordMetrics(segment, scope) {
```

Comment:
I would remove this method and tweak how it's being used in `shim/datastore-shim`